### PR TITLE
Fix manila deployment in batch deployments

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by toabctl** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4188,7 +4188,10 @@ function onadmin_batch()
                 get_novacontroller
                 oncontroller oncontroller_manila_generic_driver_setup
                 get_manila_service_instance_details
-                sed -i "s/##manila_instance_name_or_id##/$manila_service_vm_uuid/g;s/##service_net_name_or_ip##/$manila_tenant_vm_ip/g" ${scenario}
+                sed -i "s/##manila_instance_name_or_id##/$manila_service_vm_uuid/g; \
+                        s/##service_net_name_or_ip##/$manila_tenant_vm_ip/g; \
+                        s/##tenant_net_name_or_i##/$manila_tenant_vm_ip/g" \
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: typo: tenant_net_name_or_i

**Comment by toabctl** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**